src/screens/StatScreen.tsx

```diff
@@ -2,10 +2,12 @@ import React from "react";
 import { View, Text } from "react-native";
 import { globalStyles } from "../../styles";
 import DateListView from "../components/todo/DateListView";
+import ProjectProgressGraph from "../components/todo/ProjectProgressGraph";
+import ProjectProgressBarChart from "../components/todo/ProjectProgressBarChart";
 
 export const StatScreen = () => (
   <View style={globalStyles.centered}>
-    <Text>Visualisations coming soon!</Text>
+    <ProjectProgressBarChart></ProjectProgressBarChart>
```

Comment:
If a given `ReactNode` doesn't enclose any children, you can simply do
```suggestion
    <ProjectProgressBarChart/>
```
Which is a little bit more concise and adhering to convention.